torchaudio/__init__.py

```diff
@@ -11,8 +13,12 @@
     transforms,
     utils,
 )
+
 from torchaudio.backend import get_audio_backend, list_audio_backends, set_audio_backend
```

Comment:
Here, when `torchaudio.backend` is imported, the conventional backend initialization logic is executed in [`torchaudio.backend.__init__`](https://github.com/pytorch/audio/blob/main/torchaudio/backend/__init__.py).

I think it's easier to follow if the initialization logic is consolidated to one place. Either here (`torchaudio.__init__`) or in `torchaudio.backend.__init__.py`.

What do you think?